src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -21,18 +21,21 @@ interface Props {
   mutedIconTitle?: string;
   /** Title attribute for the icon when unmuted, it defaults to `Microphone` */
   unmutedIconTitle?: string;
+  /** A boolean that determines whether or not to include additional sample audio devices to the control bar */
+  appendAdditionalDevices?: boolean;
```

Comment:
Should we add sample devices to the documentation of each controls, so that builders know what they are in context of our repo? Now that you brought up sample devices, should we just say `addSampleDevices`?

Tagging @ltrung and @simmkyu for suggestion on the prop name.

Reply:
yeah I think `sample` or `fake` or something like that would be a good distinction